src/java.base/share/classes/java/util/AbstractMap.java

```diff
@@ -877,6 +877,14 @@ public String toString() {
      */
     /* non-public */ abstract static class ViewCollection<E> implements Collection<E> {
         UnsupportedOperationException uoe() { return new UnsupportedOperationException(); }
+        // convert null entry return values into NSEE
+        <T> T nsee(T entry) {
```

Comment:
```suggestion
        <T extends Map.Entry<?, ?>> T nsee(T entry) {
```
So we don't accidentally use this on map keys or values.